cheroot/errors.py

```diff
@@ -87,4 +99,26 @@ def plat_specific_errors(*errnames):
 * https://docs.microsoft.com/windows/win32/api/winsock/nf-winsock-shutdown
 """
 
-acceptable_sock_shutdown_exceptions = (BrokenPipeError, ConnectionResetError)
+
+def _is_acceptable_socket_shutdown_error(sock_err):
```

Comment:
I'd rather have this helper only handling non-specific exceptions and leave the narrowed down ones in explicit except blocks.